test/templates/tile/index.njk

```diff
@@ -99,13 +78,8 @@
     {% call tile_content({ title: data.properties.title }) -%}
     {%- endcall %}
     {%- call tile_actions() -%}
-    {{  button(
-        {
-            icon: 'overflow'
-        },
-        modifier={ block: ['light'] },
-        aria={ label: 'More' })
-    }}
+    <button class="sap-icon--overflow" aria-label="More"></button>
```

Comment:
maybe we can avoid any `fundamental-styles` classes. Instead you can put a custom class and style it like mode menu

Reply:
so you wanna me to remove the icon?

Reply:
you can have an icon but without using directly fundamental classes

Reply:
checked at Greg did the button self contained component, icons can be used directly, and it's used @import "../scss/icons/icon" to achieve it
![Screen Shot 2019-08-05 at 9 17 25 AM](https://user-images.githubusercontent.com/41170611/62467663-43949600-b762-11e9-8527-5f1398ec6fb1.png)